assets/src/blocks/ActionPageDummy/ActionPageDummyBlock.js

```diff
@@ -0,0 +1,13 @@
+const { registerBlockType } = wp.blocks;
+
+export const registerActionPageDummyBlock = () => {
+  registerBlockType('planet4-blocks/action-page-dummy', {
+    title: 'Action Page Dummy',
```

Comment:
Wouldn't we need to translate this text?

Reply:
I don't think it's necessary in this case, since the block isn't meant to be visible in the editor 🤔 It can only be added via code

Reply:
It would probably be confusing for editors to have this string to translate even though they cannot see it anywhere!